### PR TITLE
Remove withdraw functionality

### PR DESCRIPTION
**Pull request by emmabeynon**
This removes withdraw functionality for Panopticon, as this is now being handled in Publisher (see: https://github.com/alphagov/publisher/pull/528)

Trello: https://trello.com/c/xM4HRRYt/574-preparatory-work-moving-withdrawing-functionality-into-publisher-5

image

Paired with @whoojemaflip

### CODE REVIEW

**Comment by brenetic** [comment]
👍

File: features/registering_resources.feature [hunk 1]
```diff
@@ -29,8 +29,3 @@ Feature: Registering resources
     When I put a new item into panopticon whose slug is already taken
     Then I should receive an HTTP 409 response
       And the relevant artefact should not be updated
-
-  Scenario: Deleting an item
```
Comment (davidbasalla) [inline features/registering_resources.feature]: There are some corresponding step definitions in registration_steps.rb that could be removed

**Comment by davidbasalla** [comment]
Looks good, only one minor comment. Have we pretty much copied over all the tests from here to Publisher? Just wanting to make sure that we retain the same set of features/functionality

File: features/registering_resources.feature (hunk 1 above)
Comment (emmabeynon) [inline features/registering_resources.feature]: Now removed :)

**Comment by emmabeynon** [comment]
@davidbasalla We've double checked and all the relevant tests have been copied over to Publisher 👍

**Pull request merged**